.github/ISSUE_TEMPLATE/Standard-Feature-Issue-Template.md

```diff
@@ -22,31 +22,45 @@ so that < why do you need this (short description of the business value >.
 ## Notes
 < bulleted list of special notes or suggestions to the development team on the implementation of the changes. >  
 
-## Definition of Ready
+## Ready for Planning
 
 - [ ] The Issue maintains standard format for Issues and meets INVEST
-- [ ] Issue Defined and written appropriately
-- [ ] Acceptance Criteria Defined
-- [ ] User acceptance test cases defined
+- [ ] Story and Business Value complete
+- [ ] Initial Acceptance Criteria Defined
 - [ ] The person who will Approve story identified
-- [ ] Dependencies Identified
+
+## Ready for Ready to Work
+- [ ] Acceptance Criteria Refined
 - [ ] Issue size estimated by Dev Team
-- [ ] Proposed solution accepted
+- [ ] Dependencies Identified
 
-## Definition of Done
+## Ready for Review
+- [ ] Proposed solution defined and accepted by Approver
+- [ ] Code implemented and built
+- [ ] Testing implemented and passed
 
-- [ ] Issue accepted by PO
-- [ ] Issue is scored
+## Ready for Pre-Production QA
+- [ ] Peer Reviews complete with all changes tested
 - [ ] All features implemented
 - [ ] All features not yet implemented or inactive are hidden from the user
-- [ ] Peer Reviews complete	
-- [ ] Unit and regression tests are Passed
-- [ ] Manual Testing Complete and issues resolved
-- [ ] Automated tests created and run successfully
-- [ ] All blockers addressed
+- [ ] Special testing instructions noted
+- [ ] Acceptance Test cases created and reviewed by QA
 - [ ] Acceptance criteria met
-- [ ] Deployed and verified in Production
-- [ ] Ready for PO Approval
+
+## Ready for Deploy and Test
+- [ ] PreProduction QA Test Passed
```

Comment:
Do we have bug tolerance? I mean, what happens if a bug is found but is not a show stopper, in this case, not all tests will pass. I think we might add something like:

- [ ] No critical (show stopper) bugs have been found.

Reply:
In my opinion, that is part of the definition of "Passed" for the QA testing. This checkbox is saying basically "we can move to deploy only if QA tells us it is good to go" Bug tolerance is built into the definition of quality. In my opinion, there should be no bugs in the newly created Feature. Rather deal with bug tolerance as an exception case.

Reply:
Great, if there is no bug tolerance, for me it's ok then.